en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,114 @@
+    with one entry per major release line.
+    Example for an issue affecting all framework releases: First line `^3.0`, second line `^4.0`.
+    We don't know the target release version at this point, so can't set an upper constraint yet.
+    It should include all affected versions, including any unsupported release lines.
+  * **Affected component(s):** E.g. ORM, Template layer
+  * **Suggested description of the vulnerability:** Keep this short, usually the issue title only.
+    We want to retain control over editing the description in our own systems without going
+    through lengthy CVE change processes.
+  * **Reference(s):** Leave this blank. We'll send through a "Link to the advisory" as part of the publication request
+
+## Special circumstances
+
+### An issue has been reported publicly on GitHub
+
+Occasionally, some developers will open a public issue on GitHub reporting a vulnerability, either because they are unaware of the proper process or because they did not immediately grasp the security implication of the bug being reported.
+
+In those situations, the security team will assess how likely it was that the vulnerability was widely noticed based on how long the issue was opened on GitHub and how explicit the description was. If the risk of the vulnerability having been noticed is small, an owner on the Silverstripe GitHub organisation will delete the public facing issue and create a private one instead.
+
+### Handling issues on Huntr
+
+[Huntr.dev](https://huntr.dev/) is an open source vulnerability bounty platform. The security team occasionally receives vulnerability reports from Huntr. Reports from Huntr go through more or less the process as the regular reports. The security team should take the following points into consideration when working on a Huntr issue:
+
+- While you can reply via email and the response will be recorded in Huntr, who sent the response will not. To keep the discussion flowing more naturally, it is preferable to login into Huntr with your GitHub account and respond via the Huntr interface.
+- People reporting vulnerabilities via Huntr are not always very familiar with Silverstripe CMS. Don't assume that they have a lot of context on what Silverstripe CMS is used for.
+- Some Huntr reporters are just looking to make a quick buck. Review the reporter's profile to see how serious and reliable they are. Occasionally, some reporters will try to open multiple reports for the same underlying vulnerability or push for a higher CVSS score to get a bigger bounty. Do not hesitate to push back if the reporter is being unreasonable.
+- Once we "confirm" an issue in Huntr, it means that we agree the bug exist and that it's a vulnerability. Security enhancement should not be confirmed.
+- By default, Huntr will issue CVEs for issues reported through their platform. We've disabled this functionality.
+
+### A vulnerability has been reported against a dependency of Silverstripe CMS
```

Comment:
This part is brand new and aims to clarify how we approach dependency management in a security context.